lib/wallet/src/Cardano/Wallet/Write/Tx/SizeEstimation.hs

```diff
@@ -228,15 +230,14 @@ txConstraints (ProtocolParameters protocolParams) witnessTag = TxConstraints
     -- skeleton.
     marginalCostOf :: TxSkeleton -> Coin
     marginalCostOf skeleton =
-        Coin.distance
-            (estimateTxCost feePerByte empty)
-            (estimateTxCost feePerByte skeleton)
+        estimateTxCost feePerByte skeleton <\>
+        estimateTxCost feePerByte empty
```

Comment:
Theoretically a change in behaviour, but it both shouldn't matter and the new one might be saner 👍